### DISCUSSION TITLE
What is the term used to describe older versions of osu!lazer after its official release?

### DISCUSSION THREAD

**Discussion by azutake** [ppy/osu#20380] [Q&A]
Currently the term "osu!stable" is used, but once lazer is released, there will no longer be a term for the older version.
Any suggestions?
https://github.com/ppy/osu/pull/20340#discussion_r974355851

**Comment by ghost** [comment]
The term "classic" is used in the skin and mod designed to emulate these "older versions", so I think "osu!classic" would be a fitting name.

**Reply by peppy** [reply]
That's my current leaning too.
